src/api.py

```diff
@@ -4,7 +4,20 @@
 
 from dds import DDS
 
+import yaml
+
+# Configuration filename
+config_filename = "server.yaml"
+
+config = {}
+try:
+    with open(config_filename) as config_handle:
+        config = yaml.safe_load(config_handle)
+except Exception as err:
+    print("Unable to load configuration {0}: {1}".format(config_filename, err))
```

Comment:
What does print do here? Should this be logged instead? And should we treat it as a warning or a fatal error?

Reply:
Missing file deservers app.logger.warning. A parser error should be upgraded to a fatal error.